src/python/packages/amwg/amwg.py

```diff
@@ -862,13 +862,9 @@ def customizeTemplates(self, templates, data=None, varIndex=None, graphicMethod=
                 tm1.legend.y1  += 0.05
                 tm1.legend.y2   = tm1.legend.y1 + 0.01
                 if graphicMethod is not None:
-                    #line = vcs.createline('dash')
-                    #line.width = 2
-                    #line.color = ["red"]#, "blue", "salmon", "medium aquamarine", "orange", "chartreuse"]
-                    #line.type = ["dash"]#, "dot", "dash-dot", "long-dash", "solid", "dash"]
-                    #graphicMethod.line = line
-                                        
-                    graphicMethod.linecolor = 242
+                    graphicMethod.linetype = "dash"
+                    # The next repeated commands were necessary in Linux.
+                    graphicMethod.linecolor = "red"
                     graphicMethod.linewidth = 2
                     graphicMethod.linetype  = 1
```

Comment:
@doutriaux1 you have the same command (linetype) executed two times for the same type: lines 865 and 869.